addons/remove-from-profile-rows/addon.json

```
@@ -0,0 +1,107 @@
+{
+  "name": "Button for removing items from profile rows",
+  "description": "On rows like \"Favorite Projects\" on your profile, add the option to remove projects/users from the list. For example, this will unfavorite a project when used on the \"Favorite Projects\" row.",
+  "tags": ["community", "profiles"],
+  "enabledByDefault": false,
+  "versionAdded": "1.32.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["profiles", "https://scratch.mit.edu/users/*/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["profiles", "https://scratch.mit.edu/users/*/*"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Only show on the dedicated pages",
+      "id": "dedicated_pages",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Only show the button on hover",
```

Could probably be renamed:
```suggestion
      "name": "Only show remove button while hovering over items",
```